### ISSUE TITLE
Non ASCII character

### ISSUE THREAD

**Issue by kurktchiev** [splunk/splunk-operator#32]
Seems that the `,` in the Dockerfile label ends up getting reported as non ASCII.

https://github.com/splunk/splunk-operator/blob/eace63e14b84c3926d8450f8dc0a6c123bce6ced/build/Dockerfile#L13

**Comment by kalinon** [comment]
This is present in the following image:
```
docker.io/splunk/splunk-operator                                 latest                       fe27e19956d8
```

You can verify with:
```
docker inspect docker.io/splunk/splunk-operator | LC_ALL=C grep --color='auto' -P -n "[\x80-\xFF]"
```
![Screen Shot 2020-02-11 at 11 27 00 AM](https://user-images.githubusercontent.com/5629164/74256494-6d0a1d80-4cc1-11ea-830a-c05cada88bd5.png)

**Comment by mikedickey** [comment]
Merged to develop for inclusion in 0.1.0
